lib/kafo/kafo_configure.rb

```diff
@@ -166,6 +166,13 @@ def logger
     def run(*args)
       started_at = Time.now
       logger.debug("Running installer with args #{args.inspect}")
+      if config.app[:verbose]
+        log_output_message = <<~MESSAGE.strip
+          Running installer with log based terminal output at level #{config.app[:verbose_log_level].upcase}.
+          Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG.
+        MESSAGE
+        logger.notice(log_output_message)
```

Comment:
I don't know how well our tooling deals with multi-line output. Perhaps it should be this:
```suggestion
        puts "Running installer with log based terminal output at level #{config.app[:verbose_log_level].upcase}."
        puts "Use -l to set the terminal output log level to ERROR, WARN, NOTICE, INFO, or DEBUG."
```

Reply:
I want to see for myself how the actual output looks with these approaches we are discussing. I'll test it and report back shortly

Reply:
Personally, I like sticking with log based output for the consistency of it visually and code wise.

Out of curiosity, is anyone aware of other tools that do the above that we are trying to do here? I'm not saying we have to adhere to trends, but I tend to think about comparing this to what other experiences users might have and their experiences therein.

Reply:
Thanks for the feedback. Based on this discussion, I think the most visually consistent approach is to keep using `logger.notice` but split it into two lines. The latest change I pushed takes that approach.